Bundle/QueryBundle/Helper/QueryHelper.php

```diff
@@ -142,6 +142,11 @@ public function buildWithSubQuery(VictoireQueryInterface $containerEntity, Query
         //Add ORDER BY if set
         if ($orderBy = json_decode($containerEntity->getOrderBy(), true)) {
             foreach ($orderBy as $addOrderBy) {
+                // Ignore this order if one parameters is missing.
+                if ((!isset($addOrderBy['by'])) || (!isset($addOrderBy['order']))) {
+                    continue;
```

Comment:
Something specific like a "OrderByClauseException" with the $addOrderBy array as argument. it can display a message like sprintf('The orderBy clause is invalid, %s clause is missing', $missingClause)
Where $missingClause is "order" or "by" or "both" depending on which chause is missing